src/app/routes/inscription.rs

```diff
@@ -0,0 +1,182 @@
+      {children(cx)}
+    </div>
+  }
+}
+
+#[component]
+pub fn Title(cx: Scope, children: Children) -> impl IntoView {
+  view! { cx, <h3 class="text-xs lg:text-md text-gray-400 dark:text-gray-400">{children(cx)}</h3> }
+}
+
+#[component]
+pub fn Label(cx: Scope, children: Children) -> impl IntoView {
+  view! { cx,
+    <h2 class="text-xs sm:text-base lg:text-2xl text-gray-800 dark:text-gray-200 text-ellipsis overflow-hidden">
+      {children(cx)}
+    </h2>
+  }
+}
+
+#[component]
+pub fn Inscription(cx: Scope) -> impl IntoView {
+  let params = use_params_map(cx);
+  let id = move || params().get("id").cloned().unwrap_or_default();
+  let hash = id.derive_signal(cx);
+  let (fullscreen, set_fullscreen) = create_signal(cx, false);
+
+  view! { cx,
+    <div class="content">
+      {move || {
+          match &id().is_empty() {
+              true => {
+                  view! { cx, <h1 class="text-4xl text-center">"Invalid inscription id"</h1> }
+                      .into_view(cx)
+              }
+              false => {
+                  view! { cx,
+                    <div class="relative px-20 md:px-40 lg:px-60 my-10 flex items-center justify-center">
+                      <button
+                        class="group relative w-full h-full"
+                        on:click=move |_| set_fullscreen.set(true)
+                      >
+                        <Preview
+                          class="ring-4 sm:ring-8 ring-red-500 rounded-lg aspect-w-4 aspect-h-4 group-hover:brightness-100"
+                          hash
+                        />
+                        <div class="absolute right-2 top-2 sm:right-5 sm:top-5 shadow-md ease
+                        bg-white opacity-80 group-hover:opacity-100 text-gray-900 w-6 h-6 sm:w-10 sm:h-10 flex justify-center items-center p-1 sm:p-2 rounded-full">
+                          <svg
+                            xmlns="http://www.w3.org/2000/svg"
+                            fill="none"
+                            viewBox="0 0 24 24"
+                            strokeWidth=1.5
+                            stroke="currentColor"
+                            class="w-6 h-6"
+                          >
+                            <path
+                              strokeLinecap="round"
+                              strokeLinejoin="round"
+                              d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
+                            ></path>
+                          </svg>
+                        </div>
+                      </button>
+                    </div>
+                    <Container>
+                      <Title>"id"</Title>
+                      <Label>{&id()}</Label>
+                    </Container>
+                    <Container>
+                      <Title>"address"</Title>
+                      <Label>"bc1pyqz4gyra5cssfgawk6dunh4s9car9zrzpkw2l0sz4g9ym0cdxzescnf4u2"</Label>
```

Comment:
Lets hide the dummy data based on `LEPTOS_ENV` or so. 
this allows us to merge this major refactor for devs without showing false info in prod.

for dummy data use the first mainnet inscription
https://ordinals.com/inscriptions/0
https://ordinals.com/inscription/6fb976ab49dcec017f1e201e84395983204ae1a7c2abf7ced0a85d692e442799i0

Reply:
> Lets hide the dummy data based on LEPTOS_ENV

Ok, hided in `DEV` now (2e1ef53)

> for dummy data use the first mainnet inscription

Why? Mock data are still mock data.

Reply:
as an homage to casey